src/algokit/core/bootstrap.py

```diff
@@ -34,15 +35,17 @@ def bootstrap_any(project_dir: Path, *, ci_mode: bool) -> None:
         bootstrap_npm(project_dir)
 
 
-def bootstrap_any_including_subdirs(base_path: Path, *, ci_mode: bool) -> None:
+def bootstrap_any_including_subdirs(base_path: Path, *, ci_mode: bool, depth: int = 0) -> None:
```

Comment:
@robdmoore @neilcampbell these are to ensure that we dive at least 2 folders deep by default instead of 1 level that we get right now since new template structure is by default require 2 nesting levels 'src/{some_project}` and without these changes bootstrap won't get to them. We can also explore parametrizing lookup depth as a parameter you can pass to bootstrap